build/scripts/buildWebKitFromWithinXcode.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-xcodebuild -target JavaScriptCore -sdk "$SDKROOT" -configuration $CONFIGURATION ARCHS="$ARCHS" ONLY_ACTIVE_ARCH=$ONLY_ACTIVE_ARCH $DEPLOYMENT_TARGET_SETTING_NAME=${!DEPLOYMENT_TARGET_CLANG_ENV_NAME}
\ No newline at end of file
+xcodebuild -target JavaScriptCore -sdk "$SDKROOT" -configuration $CONFIGURATION ARCHS="$ARCHS" ONLY_ACTIVE_ARCH=$ONLY_ACTIVE_ARCH $DEPLOYMENT_TARGET_SETTING_NAME=${!DEPLOYMENT_TARGET_CLANG_ENV_NAME} GCC_WARN_INHIBIT_ALL_WARNINGS=YES
```

Comment:
I'd rather not do that, because Xcode relies on error messages being printed in the format they are in so that they can be picked up by the issues navigator. Besides, with `GCC_WARN_INHIBIT_ALL_WARNINGS` set, the endless drove of warnings will cease and the output should be smaller. Also, you can switch away from the output tab while the build is underway to mitigate the unresponsiveness problem.